### PR TITLE
Fix pyflakes and pep8 errors

### PR DESCRIPTION
**Pull request by jayvdb**
Use extra asserts to use previously unused variables in tests,
such as `cass` and `response`.

Fix only pyflakes errors in docs/conf.py

### CODE REVIEW

**Comment by jayvdb** [comment]
fwiw, the `.travis.yml` changes will be mostly unnecessary after #228

**Comment by kevin1024** [comment]
Looks like I caused some conflicts when merging #228.  Can you rebase?

Also, I noticed quite a few new asserts.  Is this purely for pep8 reasons?  Does flake8 require an assert per test or something?

**Comment by jayvdb** [comment]
pyflakes requires that all variables are used.
Often the tests had `with blah as cass: something` which did not access `cass`.  Adding an assert for `cass` fixes the problem.  The alternative is to remove `.. as cass:`, but I guessed it is more useful to add asserts than remove `cass` which may be useful for debugging or future asserts.

**Comment by kevin1024** [comment]
Ohh gotcha.

I think I would prefer to do the first option (remove ..as cass) since I prefer to have fewer asserts per test.  Asserts without much meaning to the test can dillute the meaning of the important assert.  I strive for a single assert per test but often don't achieve it :)

File: tests/integration/test_record_mode.py [hunk 1]
```diff
@@ -9,16 +8,21 @@ def test_once_record_mode(tmpdir):
     with vcr.use_cassette(testfile, record_mode="once"):
         # cassette file doesn't exist, so create.
         response = urlopen('http://httpbin.org/').read()
```
Comment (jayvdb) [inline tests/integration/test_record_mode.py]: should I also change lines like this to `urlopen('http://httpbin.org/').read()` , and remove the `assert response` ?

(It is very late here, so I'll pick this up tomorrow and revise it to better suit your tastes.)

Comment (kevin1024) [inline tests/integration/test_record_mode.py]: Yeah, if we're not doing anything with the variable I suppose we don't need to create it.  (BTW thanks a lot for doing this!  Having all the code be pep8 gives a nice clean feeling :) :+1:

**Comment by kevin1024** [comment]
Thanks for the cleanup!

**Pull request merged**